src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -54,6 +59,14 @@ public class CommandTestUtil {
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
 
+    public static final String TASK_NAME_TASK1 = " " + PREFIX_TASK_NAME + VALID_TASK_NAME;
+    public static final String TASK_DESCRIPTION_TASK1 = " " + PREFIX_TASK_DESCRIPTION + VALID_TASK_DESCRIPTION;
+    public static final String TASK_NAME_TASK2 = " " + PREFIX_TASK_NAME + "Read quant guide";
+    public static final String TASK_DESCRIPTION_TASK2 = " " + PREFIX_TASK_DESCRIPTION + "The green book";
+    public static final String INVALID_TASK_NAME = " " + PREFIX_TASK_NAME + "@@@@HER";
```

Comment:
Might want to add comments at the side for readability
e.g // '@' not allowed in name